cmd/gaia/contrib/gitian-keys/README.md

```diff
@@ -0,0 +1,29 @@
+## PGP keys of Gitian builders and Gaia Developers
```

Comment:
Is there a particular reason this lives under `cmd`? Other projects seem to use `proj-root/contrib`, e.g. https://github.com/bitcoin/bitcoin/tree/master/contrib/gitian-keys

Reply:
`cmd/gaia` is the future `gaia` standalone repo root directory. When I move Gaia out, I will move the whole contents of it out of there

Reply:
Thanks for the explanation. Makes sense.